salvo/README.md

```diff
@@ -38,21 +61,42 @@ YAML Example:
 remote: false
 dockerizedBenchmark: true
 environment:
-  envoyPath: 'envoy'
   outputDir: '/home/ubuntu/nighthawk_output'
   testDir: '/home/ubuntu/nighthawk_tests'
   testVersion: IPV_V4ONLY
+  envoyPath: 'envoy'
 images:
   reuseNhImages: true
   nighthawkBenchmarkImage: 'envoyproxy/nighthawk-benchmark-dev:latest'
   nighthawkBinaryImage: 'envoyproxy/nighthawk-dev:latest'
-  envoyImage: "envoyproxy/envoy:v1.16.0"
+  envoyImage: "envoyproxy/envoy:v1.21.0"
 ```
 
-In both examples above, the envoy image being tested is a specific tag. This tag can be replaced with "latest" to test the most recently created image against the previous image built from the prior tag. If a commit hash is used, we find the previous commit hash and benchmark that container.  In summary, tags are compared to tags, hashes are compared to hashes.
+`remote`: Whether to enable remote excution mode.
```

Comment:
Can we provide few more details about what is meant by remote execution? What changes when we configure this as remote? What remote environments are supported?

Reply:
Remote mode has not been implemented yet, refer to:
https://github.com/envoyproxy/envoy-perf/blob/ed970c5986f662484920a8d7b1e40c7c29a48363/salvo/src/lib/run_benchmark.py#L397-L415

Do you know any more about the original target of `remote mode`?

Reply:
I don't know what the original intent was, but maybe we can ask @abaptiste? I suspect it was related to running on VMs in CI or similar.

Either way, the remote execution mode is irrelevant for our immediate goal (documenting the local execution). We can revisit the remote mode once we will be working on running Salvo in CI.

Reply:
@mum4k Thanks, got it. I‘ll focus on local execution at this stage.

Reply:
Right, remote mode has not been implemented.  We intended to build a mode where Salvo uploads all required artifacts to a remote service where the benchmarks would be executed in parallel in isolated environments.